File: app/src/main/java/com/jiujiu/mvvmTemplate/ui/main/fragments/OrdersFragViewModel.java
package com.jiujiu.mvvmTemplate.ui.main.fragments;

import com.jiujiu.mvvmTemplate.data.DataManager;
import com.jiujiu.mvvmTemplate.ui.base.BaseViewModel;

import javax.inject.Inject;

public class OrdersFragViewModel extends BaseViewModel {
    @Inject
    public OrdersFragViewModel(DataManager dataManager) {
        super(dataManager);
    }


}

File: app/src/main/java/com/jiujiu/mvvmTemplate/ui/orders/OrdersFragViewModel.kt
package com.jiujiu.mvvmTemplate.ui.orders

import com.jiujiu.mvvmTemplate.data.DataManager
import com.jiujiu.mvvmTemplate.ui.base.BaseViewModel

import javax.inject.Inject

class OrdersFragViewModel @Inject constructor(
        dataManager: DataManager
) : BaseViewModel(dataManager)
